internal/hcs/process.go

```diff
@@ -336,26 +332,17 @@ func (process *Process) Stdio() (_ io.WriteCloser, _ io.ReadCloser, _ io.ReadClo
 
 	var stdIn, stdOut, stdErr syscall.Handle
```

Comment:
Why cant we just remove these and below use:

```go
pipes, err := makeOpenFiles([]syscall.Handle{processInfo.StdInput, processInfo.StdOutput, processInfo.StdError})
```

Reply:
Good point.